### DISCUSSION TITLE
Welcome to xml2lua Discussions!

### DISCUSSION THREAD

**Discussion by manoelcampos** [manoelcampos/xml2lua#56] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by trgjtrgj** [comment]
Hi, I downloaded XML 2 lua from Reactor on Davinci Resolve. I've been searching for something like this for a long time to convert files created in XML to lua. However, there are no instructions on how to use the plugin in davinci. Could you help me with that?

**Reply by manoelcampos** [reply]
Hello, unfortunately I don't know anything about Davinci Resolve plugins, therefore I can't help you.
